prow/gerrit/adapter/adapter.go

```diff
@@ -506,6 +510,33 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 	return nil
 }
 
+// shouldSkipProcessingChange returns true when there is no new commit or relevant commands in the comment messages
+func (c *Controller) shouldSkipProcessingChange(instance string, change client.ChangeInfo, latest client.LastSyncState) bool {
+	lastUpdate, exists := latest[instance][change.Project]
```

Comment:
This is not threadsafe, you would need to lock to use this. Rather than an unnecessary lock on the entire map, please pass the `syncTime` variable down through `processChange` since that is the value for this project.

Reply:
Done.